### PR TITLE
Copter: fix compile when precland is not selected

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: ArduCopter/switches.cpp [hunk 1]
```diff
@@ -550,6 +550,7 @@ void Copter::do_aux_switch_function(int8_t ch_function, uint8_t ch_flag)
             break;
 
         case AUXSW_PRECISION_LOITER:
+#if PRECISION_LANDING == ENABLED
```
Comment (muramura) [inline ArduCopter/switches.cpp]: I think that it is better to include case statements.
Code size will also be reduced slightly.

Comment (rmackay9) [inline ArduCopter/switches.cpp]: Re the case statements, I think we might get compiler warnings if we don't include every item in the enum.. I think that may be why Peter has left it in.

**Comment by rmackay9** [comment]
Merged!

File: ArduCopter/switches.cpp (hunk 1 above)
Comment (muramura) [inline ArduCopter/switches.cpp]: @rmackay9 Comment Thank you.
I understand.
There is such a thing.

**Pull request merged**